data collector/get_data.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import logging
+from time import sleep
+from datetime import datetime, timedelta
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import SQLAlchemyError
+from Flask_create import db
+from db_model import CoinsHist
+from db_config import DATABASE_URL, SCHEMA_NAME, API_URL
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger()
+
+engine = create_engine(DATABASE_URL)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+# fetch, parcer, 
+
+def fetch_and_store_data():
```

Comment:
Too many responsibility in one function
I'd suggest to split it:
APIClient:
  responsible on getting API data and handling HTTP errors
Parser:
  validating data
DAO:
  storing data